### PR TITLE
add termynal

### PR DESCRIPTION
**Pull request by eshaben**
This PR adds termynal as a replacement for screenshots!

### CODE REVIEW

File: material-overrides/assets/stylesheets/termynal.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+[data-termynal] {
+    width: 750px;
+    max-width: 100%;
+    background: var(--color-bg);
+    color: var(--color-text);
+    font-size: 12px;
+    font-family: 'Fira Mono', Consolas, Menlo, Monaco, 'Courier New', Courier, monospace;
+    border-radius: 4px;
+    padding: 55px 45px 35px;
```
Comment (albertov19) [inline material-overrides/assets/stylesheets/termynal.css:23]:
```suggestion
      padding: 40px 50px 20px 30px;
```

File: material-overrides/assets/stylesheets/termynal.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    --color-text: #eee;
+    --color-text-subtle: #a2a2a2;
+}
+
+[data-termynal] {
+    width: 750px;
+    max-width: 100%;
+    background: var(--color-bg);
+    color: var(--color-text);
+    font-size: 12px;
```
Comment (albertov19) [inline material-overrides/assets/stylesheets/termynal.css:20]:
```suggestion
    font-size: 13px;
```

File: material-overrides/assets/stylesheets/termynal.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    /* Set up defaults and ensure empty lines are displayed. */
+    content: '';
+    display: inline-block;
+    vertical-align: middle;
+}
+
+[data-ty="input"]:before,
+[data-ty-prompt]:before {
+    margin-right: 0.75em;
+    color: #189b9b;
```
Comment (albertov19) [inline material-overrides/assets/stylesheets/termynal.css:69]:
```suggestion
    color: #189b9b;
    font-size: 14px;
    content: 'tanssi@ubuntu-jammy: ~$';
```

File: material-overrides/assets/stylesheets/termynal.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+[data-ty="input"]:before {
```
Comment (albertov19) [inline material-overrides/assets/stylesheets/termynal.css:72]:
```suggestion
```

File: material-overrides/assets/stylesheets/termynal.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    content: 'tanssi@ubuntu-jammy:~$';
```
Comment (albertov19) [inline material-overrides/assets/stylesheets/termynal.css:73]:
```suggestion
```

File: material-overrides/assets/stylesheets/termynal.css [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+}
```
Comment (albertov19) [inline material-overrides/assets/stylesheets/termynal.css:74]:
```suggestion
```

**Review by albertov19 — CHANGES_REQUESTED** [review]
Hey, checked some other configurations that I liked better. Let me know what you think :)

**Comment by eshaben** [comment]
Screenshot 2024-01-12 at 4 55 49 PM

**Comment by eshaben** [comment]
If this looks good, I'll apply similar changes to moonbeam 🙂

**Review by albertov19 — APPROVED** [review]
LGTM!!

**Pull request merged**